recipe/meta.yaml

```diff
@@ -29,6 +29,9 @@ requirements:
     - sqlite
     - libtiff
     - libcurl
+    - gtest  # [not win]
```

Comment:
This is a different "unknown" issue. If "gtest" is provided for windows there are some dll errors. Skipping it for windows forces googletest to be downloaded for "internal" use.